lisa/tools/__init__.py

```diff
@@ -6,4 +7,4 @@
 from .ntttcp import Ntttcp
 from .uname import Uname
 
-__all__ = ["Echo", "Gcc", "Git", "Lscpu", "Make", "Ntttcp", "Uname"]
+__all__ = ["Cat", "Echo", "Gcc", "Git", "Lscpu", "Make", "Ntttcp", "Uname"]
```

Comment:
Agreed.  I never said anything about `Echo` because it's a useful example on how to write a new tool, but if we can use Python APIs, we should just use them.

Reply:
Those operations support both local and remote ssh nodes. I thought a lot, if lisa dispatches python code to remote node, and then run it. But it looks more complex on environment setup, so most python native file, process operations cannot be used.